install_config/upgrading/os_upgrades.adoc

```diff
@@ -0,0 +1,27 @@
+[[install-config-upgrading-os-upgrades]]
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+When you need to upgrade the Operating System, either changing OS
+versions or just updates to the system software, you need to be aware
+that this can have an impact on the {product-title} software running
+on those machines.
+
+In particular, these updates can affect the `iptables` rules that
```

Comment:
iptables isn't the only source of problems... if OVS gets updated, then the ovs db will get restarted, which also causes problems. Although the solution is basically the same as in the iptables case: restart the processes that care.

It looks like the docker daemon gets restarted on docker update too, which will kill any running pods.

Maybe we should just say "if you want to update, stop the node, update, and reboot"?